packages/vector_graphics_compiler/test/end_to_end_test.dart

```diff
@@ -458,8 +458,8 @@ class OnRadialGradient {
         focalX,
         focalY,
         Object.hashAll(colors),
-        Object.hashAll(offsets ?? []),
-        Object.hashAll(transform ?? []),
+        Object.hashAll(offsets ?? Float32List(0)),
```

Comment:
This isn't a const object so it will be different for the otherwise equal object

Reply:
fiddlesticks

Reply:
(fixed it to use `<double>[]`)